docker-compose.yml

```diff
@@ -71,32 +72,25 @@ services:
     networks:
       - x11
 
-  foundry:
```

Comment:
Can we let this docker-compose work as before so that metamast/ foundry tests run like before so that we don't break them?

Reply:
For our use case we can create another docker compose file, or add our stuff in this docker-compose and run docker compose commands with parameters to trigger metamast or keplr based tests.